cassandra/src/cassandra/core.clj

```diff
@@ -136,6 +136,13 @@
                    (set/difference @(:decommissioned test))
                    shuffle))))
 
+(defn seed-nodes
+  "Get a list of seed nodes"
+  [test]
+  (if (= (:rf test) 1)
+    (first (:nodes test))
+    (take (dec (:rf test)) (:nodes test))))
```

Comment:
Just a question. Is it OK to assume the seed nodes come first ?

Reply:
Seed nodes are configured by this function in `configure!`.